### PR TITLE
Iss916 - Use SIFGraph service

### PR DESCRIPTION
**Pull request by jvwong**
re #916 and also  #1070

I just assume the nodes are HGNC symbols - this won't fly with CHEBI but I'm restricting patterns right now

### CODE REVIEW

**Comment by maxkfranz** [comment]
For the upcoming release, I suppose #1070 is the more pressing issue?  How does the server perform with the new N=25 restriction?  Still too slow?

**Comment by d2fong** [comment]
http://appsbeta.pathwaycommons.org/interactions?source=FGF9 seems ok with 25 nodes
http://appsbeta.pathwaycommons.org/interactions?source=ROCK1 doesn't seem to be affected by network size.

**Comment by d2fong** [comment]
You are going to have to update the tests:
https://travis-ci.org/PathwayCommons/app-ui/jobs/429666928

/test/server/interactions/interactions-network-generation-test.js

File: test/server/interactions/interactions-network-generation-test.js [hunk 1]
```diff
@@ -150,32 +95,4 @@ describe('Interactions network to cy.js conversion', function(){
 
   });
 
-  it('filters nodes with 0 degree', function(){
```
Comment (d2fong) [inline test/server/interactions/interactions-network-generation-test.js]: I still think it is useful to have a test showing that we filter degree 0 nodes.

Comment (maxkfranz) [reply test/server/interactions/interactions-network-generation-test.js]: I agree it's better to catch the edgecases as best as possible with the tests

**Pull request merged**